browser/playlist/playlist_tab_helper.cc

```diff
@@ -78,6 +78,26 @@ void PlaylistTabHelper::RemoveItems(std::vector<mojom::PlaylistItemPtr> items) {
   }
 }
 
+void PlaylistTabHelper::MoveItems(std::vector<mojom::PlaylistItemPtr> items,
+                                  mojom::PlaylistPtr target_playlist) {
+  for (const auto& item : items) {
+    DCHECK_EQ(item->parents.size(), 1u);
```

Comment:
Question about this check.
AFAIK, item could be belonged to different playlists. so this check makes me confusing.

Found this - https://github.com/brave/brave-core/pull/19383/files#diff-a9be76421abd17e420a9e4553ca8c5805fc4a3288129d967616b13b9465d327eR123-R124

Reply:
Yeah, exactly. This version should be used only when the DCHECK() condition is met. In case an item belongs to multiple items, this version shouldn't be called.

Reply:
Maybe this one should be a `CHECK`? Will things break if this one is accidentally used?

Reply:
Yep, `CHECK` makes sense. Changed and added message.